spec/lib/facilities/ppms/client_spec.rb

```diff
@@ -1,89 +1,183 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
-module Facilities
-  RSpec.describe PPMS::Client do
-    let(:bbox_bounds) { [-79, 38, -77, 39] }
 
-    it 'is an PPMS::Client object' do
-      expect(described_class.new).to be_an(PPMS::Client)
-    end
+RSpec.describe Facilities::PPMS::Client do
+  let(:params) do
+    {
+      address: 'South Gilbert Road, Chandler, Arizona 85286, United States',
+      bbox: [-112.54, 32.53, -111.04, 34.03]
+    }.with_indifferent_access
+  end
 
-    regex_matcher = lambda { |r1, r2|
-      r1.uri.match(r2.uri)
-    }
+  it 'is an PPMS::Client object' do
+    expect(described_class.new).to be_an(Facilities::PPMS::Client)
+  end
 
-    describe 'route_fuctions' do
-      context 'with an http timeout' do
-        before do
-          allow_any_instance_of(Faraday::Connection).to receive(:get).and_raise(Faraday::TimeoutError)
-        end
+  context 'with an http timeout' do
+    before do
+      allow_any_instance_of(Faraday::Connection).to receive(:get).and_raise(Faraday::TimeoutError)
+    end
```

Comment:
Indentation diff makes it harder to see on GitHub, but is this a before for a single example? If it is I'd consider just making it part of the example

Reply:
it is, I'll move it into the `it` block